playbooks/spark-integration-test-kubeadm-k8s/run.yaml

```diff
@@ -0,0 +1,28 @@
+- hosts: all
+  become: yes
+  roles:
+    - create-single-k8s-cluster-with-kubeadm
+  tasks:
+    - name: Run integration tests of Spark deployed on k8s cluster
+      shell: |
+        set -ex
+        export KUBECONFIG=/etc/kubernetes/admin.conf
+        # install java8
+        apt-get install openjdk-8-jdk -y
+        sed -i -e '/127.0.0.1/ s/\(localhost\)/'$(hostname)' \1/' /etc/hosts
+
+        # Create required account in k8s
+        kubectl create serviceaccount spark
+        kubectl create clusterrolebinding spark-role --clusterrole=edit --serviceaccount=default:spark --namespace=default
+
+        export JAVA_HOME=$(dirname $(dirname $(update-alternatives --list javac)))
```

Comment:
please move setting JAVA_HOME to the next line of installing openjdk to avoid some java calling in the middle, maybe not now, but we have to reduce the difficulty of future refactor

Reply:
done